controller.php

```diff
@@ -2,6 +2,7 @@
 
 namespace Concrete\Package\MdVisibilityAttribute;
 
+use Concrete\Core\Asset\AssetList;
```

Comment:
@SashaMcr 
I think you no longer need the select2 CSS & JS. Please remove those files and this line.

Reply:
@biplobice 

cleared